### PR TITLE
Use OpenShift 4 and OpenShift 3 also for RHEL7.

### PR DESCRIPTION
**Pull request by phracek**
RHEL7 container images can be tested on OpenShift 4 as on OpenShift 3.
This pull request solves this issue.

Let's use RHEL-7-LatestUpdate for RHEL7 machines.

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Review by zmiklank — COMMENTED** [review]
Thanks.

I would rather split this into two commits (OCP stuff in one commit and compose change in second), but if you would like to merge it as it is I am OK with that too.

File: export_variables.sh [hunk 1]
```diff
@@ -18,16 +18,14 @@ if [ -z "$test_case" ] || [ "$test_case" = "container" ] ; then
   tmt_plan_suffix="-docker"
   context_suffix=""
   test_name="test"
-elif [ "$test_case" = openshift ] ; then
-  if [ "$os_test" = "centos7" ] || [ "$os_test" = "rhel7" ] ; then
+elif [ "$test_case" = "openshift" ] ; then
```
Comment (jamacku) [inline export_variables.sh:21]: I would suggest using `case`  instead of `if elif etc.`.

Comment (phracek) [reply export_variables.sh]: Done

**Review by jamacku — APPROVED** [review]
Thank you @phracek LGTM

File: export_variables.sh [hunk 2]
```diff
@@ -5,29 +5,33 @@ private_ranch="rhel7 rhel8 rhel9 rhel9-unsubscribed"
...
-  if [ "$os_test" = "centos7" ] || [ "$os_test" = "rhel7" ] ; then
-    context_suffix=" - OpenShift 3"
-    tmt_plan_suffix="-openshift-3"
-    test_name="test-openshift-3"
-  else
-    context_suffix=" - OpenShift 4"
-    tmt_plan_suffix="-openshift4"
-    test_name="test-openshift-4"
-  fi
+if [ -z "$test_case" ] ; then
```
Comment (zmiklank) [inline export_variables.sh:16]: This condition is not valid. If `$test_case` is not set we want to fall to "container" case. If `$test_case` is "container" we fall to "container" case too.
If `$test_case` is "openshift" or "openshift-4" we want to fall to respective cases.

Comment (phracek) [reply export_variables.sh:17]: if test_case we want to fall to error. But the condition is wrong, you are right. This should be fixed by https://github.com/sclorg/tfaga-wrapper/pull/4/commits/a412952d3fe23c7a654186d2392d2eb8a6654c39.

Comment (zmiklank) [reply export_variables.sh:17]: Yes, this seems too as a valid solution, thanks for fixing this.

**Review by zmiklank — APPROVED** [review]

**Pull request merged**